vortex-datafusion/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 //! Connectors to enable DataFusion to read Vortex data.
 
+#![allow(clippy::nonminimal_bool)]
```

Comment:
Why do we need this?

Reply:
clippy tries to "simplify" the condition around which types are supported and make into `!(!condition_a && !condition_b)` which is actually harder to parse IMO